scripts/start.sh

```diff
@@ -75,5 +75,5 @@ password: ${ADMIN_PASSWORD}
 EOL
 
 printf "\e[0;32m*****STARTING SERVER*****\e[0m\n"
-echo "${STARTCOMMAND}"
-su steam -c "${STARTCOMMAND}"
```

Comment:
This ensures that the entire command string is correctly interpreted as a single argument to bash.